content/en/blog/_posts/2022-10-03-introducing-kueue.md

```diff
@@ -0,0 +1,205 @@
+- [Suspend field](/docs/concepts/workloads/controllers/job/#suspending-a-job), which allows Kueue to signal to the job controller 
+  when to start or stop a job.
+- [Mutable scheduling directives](/docs/concepts/workloads/controllers/job/#mutable-scheduling-directives), which allows Kueue to 
+  update a Job's `.spec.template.spec.nodeSelector` before starting the Job. This way, Kueue can control Pod placement while still
+  delegating to kube-scheduler the actual pod-to-node scheduling.
+
+Note that any custom job API can be managed by Kueue if that API offers the above two capabilities.
+
+### Resource model
+Kueue defines new APIs to address the requirements mentioned at the beginning of this post. The three main APIs are:
+
+- ResourceFlavor: a cluster-scoped API to define resource flavor available for consumption, like a GPU model. At its core, a ResourceFlavor is 
+  a set of labels that mirrors the labels on the nodes that offer those resources.
+- ClusterQueue: a cluster-scoped API to define resource pools by setting quotas for one or more ResourceFlavor. 
+- LocalQueue: a namespaced API for grouping and managing single tenant jobs. In its simplest form, a LocalQueue is a pointer to the ClusterQueue 
+  that the tenant (modeled as a namespace) can use to start their jobs.
+
+For more details, take a look at the [API concepts documentation](https://sigs.k8s.io/kueue/docs/concepts). While the three APIs may look overwhelming, 
+most of Kueue’s operations are centered around ClusterQueue; the ResourceFlavor and LocalQueue APIs are mainly organizational wrappers.
+
+### Example use case
+Imagine the following setup for running batch workloads on a Kubernetes cluster on the cloud: 
+- You have [cluster-autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) installed in the cluster to automatically
+  adjust the size of your cluster.
+- There are two types of autoscaled node groups that differ on their provisioning policies: spot and on-demand. The nodes of each group are 
+  differentiated by the label `instance-type=spot` or `instance-type=ondemand`.
+  Moreover, since not all jobs can tolerate running on spot nodes, the nodes are tainted with `spot=true:NoSchedule`.
+- To strike a balance between cost and resource availability, imagine you want jobs to use up to 1000 cores of on-demand nodes, then use up to
+  2000 cores of spot nodes.
+
+As an admin for the batch system, you define two ResourceFlavors that represent the two types of nodes:
+
+```yaml
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: ondemand
+  labels:
+    instance-type: ondemand 
+---
+apiVersion: kueue.x-k8s.io/v1alpha2
+kind: ResourceFlavor
+metadata:
+  name: spot
+  labels:
+    instance-type: spot
+taints:
+- effect: NoSchedule
+  key: spot
+  value: "true"
+```
+
+Then you define the quotas by creating a ClusterQueue as follows:
+```yaml
+---
```

Comment:
I think we can remove this line, as there is only one object

Reply:
done.